index.js

```
@@ -0,0 +1,110 @@
+    const width = canvas.width
+    const height = canvas.height
+
+    // Create the worker instance
+    const worker = new Worker("worker.js", { type: "module" })
+    // Listen for messages from the worker
+    worker.addEventListener("message", (e) => {
+      const { command } = e.data
+      if (command === "progress") {
+        const { progress } = e.data
+        loadingOverlay.style.display = "flex"
+        loadingBar.style.width = progress + "%"
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)